mage_ai/io/base.py

```diff
@@ -40,6 +45,48 @@ def __init__(self, verbose=False) -> None:
         self.verbose = verbose
         self.printer = VerbosePrintHandler(f'{type(self).__name__} initialized', verbose=verbose)
 
+    def _enforce_limit(self, query: str, limit: int = QUERY_ROW_LIMIT) -> str:
+        """
+        Modifies SQL SELECT query to enforce a limit on the number of rows returned by the query.
+        This method is currently supports PostgreSQL syntax, which means it can be used with
+        PostgreSQL, Amazon Redshift, Snowflake, and Google BigQuery.
+
+        Args:
+            query (str): The SQL query to modify
+            limit (int): The limit on the number of rows to return.
+
+        Returns:
+            str: Modified query with limit on row count returned.
+        """
+        if query[:6].lower() != 'select':
```

Comment:
There could be empty lines or comments at the beginning.

Reply:
There could also be WITH at the beginning of the query.

Reply:
Switched to the method below, this functionality no longer exists.